console/src/components/Services/Events/Common/Components/EventsTable.tsx

```diff
@@ -72,22 +141,59 @@ const EventsTable: React.FC<Props> = props => {
       accessor: column,
     };
   });
-  const rowsFormatted = rows.map(r => {
-    const formattedRow = Object.keys(r).reduce((fr, col) => {
+
+  const cancelButtonHandler = (row: {
```

Comment:
out, create handlers in `PendingEvents.tsx` , and pass:
`onCancelEvent` prop to:
```
  const renderRows: FilterRenderProp = (
    rows,
    count,
    filterState,
    setFilterState,
    runQuery
  ) => (
    <EventsTable
      rows={rows}
      count={count}
      filterState={filterState}
      setFilterState={setFilterState}
      runQuery={runQuery}
      columns={[
        'id',
        'status',
        'scheduled_time',
        'created_at',
        'tries',
        'actions',
      ]}
      identifier={triggerName}
      onCancelEvent={}
    />
  );
```